workflow/automation/estimation/estimate_cybershake.py

```diff
@@ -11,10 +11,10 @@
 from argparse import ArgumentParser
 from typing import List
 
-from shared_workflow.platform_config import platform_config
+from workflow.automation.platform_config import platform_config
 import qcore.constants as const
 from qcore import shared, srf, utils
-from estimation import estimate_wct
+import estimate_wct
```

Comment:
Does this automatically import from the adjacent file? Or do we need to add in the module path?